crates/core/src/client/tests/mod.rs

```diff
@@ -143,6 +143,7 @@ async fn test_simulate_transaction() {
         max_fee: FieldElement::ZERO,
         nonce,
         signature: vec![],
+        is_query: true,
```

Comment:
what does this is_query do? it sets the tx version to not be replayable in production?

Reply:
Indeed, I think the is_query sets the chain_id of the tx to 2^128 - 1 or something similiar, which avoids replays.